src/java/org/apache/cassandra/tools/nodetool/Import.java

```diff
@@ -80,6 +80,16 @@ public class Import extends NodeToolCmd
             description = "Copy data from source directories instead of moving them")
     private boolean copyData = false;
 
+    @Option(title = "fail_on_missing_index",
+            name = {"-f", "--fail-on-missing-index"},
+            description = "Fail import when SSTables do not contain SAI components")
+    private boolean failOnMissingIndex = false;
+
+    @Option(title = "no_validate_index_checksum",
+            name = {"-i", "--no-validate-index-checksum"},
+            description = "Do not validate checksum of indexes")
```

Comment:
```suggestion
            description = "Skip SSTable-attached index checksum validation")
```
Works w/ Doug's naming suggestion and is a tiny bit more specific (as I'm guessing this is only ever going to apply to SAI-like indexes that are SSTable-attached)